setup.py

```diff
@@ -173,6 +176,7 @@ def get_exts_for(name):
         "attrs>=19.2.0",
         "six>=1.12.0",
         "pep562; python_version<'3.7'",
+        "sq-native>=1.0",
```

Comment:
in what environments does this need to be built from source?

I am guessing alpine? what are the build dependencies that are needed?

Reply:
We discussed this a bit offline, Alpine is supported.

We need to be sure our CI suite is testing installation on the various platforms we want to support.

Reply:
Alpine will download the source package but there is no compilation involved, the shared library is bundled in the package until we can use [PEP 656](https://www.python.org/dev/peps/pep-0656/). If the platform is not supported the source package will act as a stub. Maybe we can add some smoke tests for different platforms to make sure there is no installation issues.

Reply:
> Maybe we can add some smoke tests for different platforms to make sure there is no installation issues.

Yes please! This is a great idea.